domain/test/usecases/myspace/copy_multiple_files_to_my_space_interactor_test.dart

```diff
@@ -0,0 +1,142 @@
+      _copyMultiplesFilesToMySpaceInteractor = CopyMultipleFilesToMySpaceInteractor(copyToMySpaceInteractor);
+    });
+
+    test('copy multiples files to my space interactor should return success with valid data', () async {
+      when(documentRepository.copyToMySpace(
+          CopyRequest(workGroupDocument1.workGroupNodeId.uuid, SpaceType.sharedSpace, contextUuid: workGroupDocument1.sharedSpaceId.uuid)))
+      .thenAnswer((_) async => [document1]);
+
+      when(documentRepository.copyToMySpace(
+          CopyRequest(workGroupDocument2.workGroupNodeId.uuid, SpaceType.sharedSpace, contextUuid: workGroupDocument2.sharedSpaceId.uuid)))
+      .thenAnswer((_) async => [document2]);
+
+      final result = await _copyMultiplesFilesToMySpaceInteractor.execute(workGroupNodes: [workGroupDocument1, workGroupDocument2]);
+      final resultsList = result
+          .map((success) => (success as CopyMultipleToMySpaceAllSuccessViewState).resultList)
+          .getOrElse(() => []);
+      expect(resultsList, [Right<Failure, Success>(CopyToMySpaceViewState([document1])), Right<Failure, Success>(CopyToMySpaceViewState([document2]))]);
+    });
+
+    test('copy multiples files to my space interactor should return success with some failures', () async {
+      final exception = DocumentNotFound();
+      when(documentRepository.copyToMySpace(
+          CopyRequest(workGroupDocument1.workGroupNodeId.uuid, SpaceType.sharedSpace, contextUuid: workGroupDocument1.sharedSpaceId.uuid)))
+      .thenThrow(exception);
+
+      when(documentRepository.copyToMySpace(
+          CopyRequest(workGroupDocument2.workGroupNodeId.uuid, SpaceType.sharedSpace, contextUuid: workGroupDocument2.sharedSpaceId.uuid)))
+      .thenAnswer((_) async => [document2]);
+
+      final result = await _copyMultiplesFilesToMySpaceInteractor.execute(workGroupNodes: [workGroupDocument1, workGroupDocument2]);
+      final resultsList = result
+          .map((success) => (success as CopyMultipleToMySpaceHasSomeFilesViewState).resultList)
+          .getOrElse(() => []);
+      expect(resultsList, [Left<Failure, Success>(CopyToMySpaceFailure(exception)), Right<Failure, Success>(CopyToMySpaceViewState([document2]))]);
+    });
+
+    test('copy multiples files to my space interactor should return success with one document', () async {
+      when(documentRepository.copyToMySpace(
+          CopyRequest(workGroupDocument1.workGroupNodeId.uuid, SpaceType.sharedSpace, contextUuid: workGroupDocument1.sharedSpaceId.uuid)))
+      .thenAnswer((_) async => [document1]);
+
+      final result = await _copyMultiplesFilesToMySpaceInteractor.execute(workGroupNodes: [workGroupDocument1]);
+      final documentsList = result
+          .map((success) => (success as CopyToMySpaceViewState).documentsList)
+          .getOrElse(() => []);
+      expect(documentsList, [document1]);
+    });
+
+    test('copy multiples files to my space interactor should return success with folder and document', () async {
```

Comment:
IMO, we not support it, please check the real behavior: what happen in MySpace with that?